WordPress/Classes/Utility/WPMediaEditor.swift

```diff
@@ -6,11 +6,19 @@ import Gridicons
  Displays the Media Editor with our custom styles and tracking
  */
 class WPMediaEditor: MediaEditor {
+    var alreadyPublishedImage: Bool = false {
+        didSet {
+            if alreadyPublishedImage {
+                hub.doneButton.setTitle(Constants.doneLabel, for: .normal)
+            }
+        }
+    }
+
     override var styles: MediaEditorStyles {
         get {
             return [
                 .insertLabel: NSLocalizedString("Insert %@", comment: "Button title used in media editor. Placeholder will be the number of items that will be inserted."),
-                .doneLabel: NSLocalizedString("Done", comment: "Done editing an image"),
+                .doneLabel: Constants.doneLabel,
```

Comment:
Since you moved the doneLabel to the constants, shouldn't you also move the other `NSLocalizedString`'s?